canonical-web-and-design/canonicalwebteam.templatefinder
# The code for templatefinder

This creates version 0.1.0 our of code copied from https://github.com/canonical-web-and-design/canonical.com/tree/41442342f730aea9c8f6bd90e40c67ecec56fd4d/templatefinder.

Although I made a couple of changes:

- Removed `pyproject.toml` for now, as I couldn't get it to install properly from a branch with pip if that was present
- Moved `tests` directory to the top level

## QA

``` bash
git clone -b templatefinder-module git@github.com:nottrobin/canonical.com.git canonical.com-tftest
cd canonical.com-tftest
./run
```

Or just look at the demo of that branch: https://canonical-com-canonical-web-and-design-pr-9.run.demo.haus/

Check the site runs fine.
